src/components/Map/index.tsx

```diff
@@ -0,0 +1,119 @@
+import { BikeRentalStation, Scooter } from '@entur/sdk'
+import React, { useState } from 'react'
+
+import ReactMapGL, { Marker } from 'react-map-gl'
+
+import PositionPin from '../../assets/icons/positionPin'
+import ScooterOperatorLogo from '../../assets/icons/scooterOperatorLogo'
+
+import { DEFAULT_ZOOM } from '../../constants'
+import { useSettingsContext } from '../../settings'
+import { StopPlaceWithDepartures } from '../../types'
+
+import BikeRentalStationTag from './BikeRentalStationTag'
+import StopPlaceTag from './StopPlaceTag'
+
+import './styles.scss'
+
+const MapView = ({
+    stopPlaces,
+    bikeRentalStations,
+    scooters,
+    walkTimes,
+    interactable,
+}: Props): JSX.Element => {
+    const [settings, { setZoom }] = useSettingsContext()
```

Comment:
Eg synst ikkje denne komponent bør vere avhengig av `settings`. Då vil eg heller at den skal ta zoom, latitude, longitude som props. Då blir den lettare å gjenbruke.

Reply:
Enig

Reply:
Skulle gjerne hatt `useSettingsContext` heilt ut frå denne komponenten. Ha heller ein `onZoomChange`-prop og la forelderkomponenten bestemme om `setZoom` skal settast i settings.

Dersom man i framtida vil ha ein visningstype med to kart ved sida av kvarandre, så ville man gjerne brukt denne -komponenten. Då kan det bli rart om zooming i det eine kartet fører til at "global zoom" blir satt, sånn at det andre kartet også blir zooma inn/ut. Derfor er det fint om man kan kontrollere dette utanfor komponenten, og komponenten blir dum og deilig.